packages/editor/src/components/editor-history/redo.js

```diff
@@ -41,4 +41,14 @@ function EditorHistoryRedo( props, ref ) {
 	);
 }
 
+/** @typedef {import('react').Ref<HTMLElement>} Ref */
+
+/**
+ * Renders the redo button for the editor history.
+ *
+ * @param {Object} props - Props.
+ * @param {Ref}    ref   - Reference with the element.
```

Comment:
Last comment, sorry about that. Let's update both Ref descriptions to `Forwarded ref`